app/build.gradle

```diff
@@ -163,23 +162,26 @@ dependencies {
     implementation "io.reactivex.rxjava2:rxandroid:$rxAndroidVersion"
     implementation "io.reactivex.rxjava2:rxjava:$rxJavaVersion"
 
+    def iconifyVersion = "2.2.2"
```

Comment:
Why did you add this one here?

Reply:
It's not used in core, so it doesn't actually need to be in the root gradle.

Reply:
Okay, I think it would be better to not use a variable, then.

Reply:
It's right next to its usage, I think it being here would be best.

Reply:
I think it is confusing to have those variables in different locations. When changing something, you never know where the variable is located, just from looking at the library `implementation` clause. Please either move it to the other variables or remove the variable and specify the version directly in the `implementation` clause.